### PR TITLE
specify kneard-ctl in active script

### LINKED ISSUE #737: kneard-ctl: command not found - when running mgr `upgrade`
```
building the system configuration...
copying 0 paths...
updating GRUB 2 menu...
activating the configuration...
setting up /etc...
/nix/store/hn86g3jr1zxlm5r9ay5wfj0k387risbv-nixos-system-validator-00-23.05.20230510.f219df9/activate: line 180: kneard-ctl: command not found
Running as unit: nixos-upgrade.service
reloading user units for root...
```

### PR DESCRIPTION
**Pull request by yanganto**
fix #737

### CODE REVIEW

File: nix/modules/single-node-validator/default.nix [hunk 1]
```diff
@@ -12,7 +13,7 @@
     git_commit_date = "${self.lastModifiedDate}"
   '';
   system.activationScripts.nixos-upgrade = ''
-    ${config.systemd.package}/bin/systemd-run --collect --unit nixos-upgrade echo level=info message="kneard node updated" $(kneard-ctl system-info --inline)
+    ${config.systemd.package}/bin/systemd-run --collect --unit nixos-upgrade echo level=info message="kneard node updated" $(${kneard}/bin/kneard-ctl system-info --inline)
```
Comment (Mic92) [inline nix/modules/single-node-validator/default.nix:16]: Why can you not use self here.

**Comment by yanganto** [comment]
@Mic92 Please approve this, or else the bot will refuse to merge. Thanks :pray:

**Comment by yanganto** [comment]
@paulmattei Could you approve this?  Mic trigger bors to merge without approval so the bot fail. Such that we can close this.

**Review by paulmattei — APPROVED** [review]

**Comment by yanganto** [comment]
bors merge

**Pull request merged**